### ISSUE TITLE
randomSeed

### ISSUE THREAD

**Issue by qcmv22** [Skuldur/Classical-Piano-Composer#32]
hi,

 I prefer to work with google Colaboratory online ...So, can I use your code to do a Jupyter notebook there ...  Should  i quote you or you prefer not.

Also, I want to render the same training with different epochs, like the 10th , 20th  , 40th , ...  but with the same randomSeed.

On the predict.py 
I put  numpy.random.seed(2)   overthere
 def generate_notes(model, network_input, pitchnames, n_vocab):

and  I  probably need another  randomSeed for tensorflow. 
Can I put it right after the import like:

    import tensorflow as tf
    tf.random.set_seed(1)

Best.
